src/main/java/jenkins/metrics/impl/TimeInQueueAction.java

```diff
@@ -158,6 +160,27 @@ public long getQueuingTimeMillis() {
         return total;
     }
 
+     /**
+     * Returns the total time this {@link Run} spent queuing, including the time spent by subtasks. This is the sum
+     * of {@link #getQueuingDurationMillis()} plus all the {@link SubTaskTimeInQueueAction#getQueuingDurationMillis()}.
+     *
+     * @return the total time this {@link Run} spent queuing
+     */
```

Comment:
This doc string is not accurate. It should be updated to reflect what the actual method is doing.